src/azure-cli/azure/cli/command_modules/servicefabric/custom.py

```diff
@@ -685,6 +663,63 @@ def update_cluster_reliability_level(cmd,
     return LongRunningOperation(cli_ctx)(update_cluster_poll)
 
 
+def validate_arguments(*args):
+    for arg in args:
+        if arg == None or arg == False:
+            raise CLIError(
+                'Invalid arguments. Please run command: `az sf cluster update -h` for help message and examples of how to use the command')
+
+def validate_none_arguments(*args):
+    for arg in args:
+        if arg is not None and arg != False:
+            raise CLIError(
+                'Invalid arguments. Please run command: `az sf cluster update -h` for help message and examples of how to use the command')
```

Comment:
you could show which is the invalid argument in the error message

Reply:
updated